outputs.tf

```diff
@@ -18,11 +18,20 @@ output "vision_url" {
   description = "IBM Visual Insights URL"
 }
 
+output "vision_username" {
+  value = "admin"
```

Comment:
Please add a note explaining that this is hard-coded, something like
`#the admin always exists on first startup for trial editions of the application, so we hard-code it here for convenience of the user. `